src/vs/workbench/parts/terminal/common/terminal.ts

```diff
@@ -67,6 +67,7 @@ export interface ITerminalConfiguration {
 	commandsToSkipShell: string[];
 	cwd: string;
 	confirmOnExit: boolean;
+	envVars: object;
```

Comment:
Let's call this `env`, that should be descriptive enough and it matches the `env` command.